docs/setup/settings.asciidoc

```diff
@@ -313,7 +313,7 @@ and the tile layer configured by `map.tilemap.url` are available in <<maps, Maps
  | Set to `true` to proxy all <<maps, Maps application>> Elastic Maps Service
 requests through the {kib} server. *Default: `false`*
 
-| [[regionmap-settings]] `map.regionmap:`
+| [[regionmap-settings]] `map.regionmap:` {ess-icon}
  | Specifies additional vector layers for
 use in <<maps, Maps>> visualizations. Supported on {ece}. Each layer
```

Comment:
Here is a setting this is also supported on ECE.

Reply:
Thank you, @KOTungseth! I did a search through the source and added the new `{ece-icon}` wherever the text said `Supported on {ece}.` When I happened to talk to @gchaps today we also agreed that we should remove the text, leaving only the icon. (If something is supported on both ECE and ESS, you get two icons, but only a single tooltip is ever displayed at any one time.)

I'll mark the other related comments as resolved after I post this reply, so that you don't get a slew of replies that basically say the same thing.